.env.sample

```
@@ -1,5 +1,5 @@
-REACT_APP_API_URL=http://localhost:4000
 NODE_ENV=development
-MONGO_URL=mongodb://localhost:27017/erxes
+MONGO_URL=mongodb://localhost:27017/erxes?directConnection=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MONGO_URL key should go before the NODE_ENV key

(UnorderedKey)